org.alloytools.alloy.core/src/main/java/edu/mit/csail/sdg/ast/Expr.java

```diff
@@ -40,6 +40,8 @@
  * <b>Invariant:</b> type==EMPTY iff errors.size()>0 <br>
  * <b>Invariant:</b> mult==0 || mult==1 || mult==2 <br>
  * <b>Invariant:</b> weight>0
+ *
+ * @modified Eduardo Pessoa, Nuno Macedo // [HASLab] electrum-temporal
```

Comment:
basically adds convenient methods to create temporal formulas (unary always, eventually, after, once, historically, before and binary until, releases, since, triggered) and expressions (primes); also updates `Decl` creations which are always immutable (null `isVar`) in this context